rrmngmnt/network.py

```diff
@@ -205,6 +205,24 @@ def find_default_gw(self):
         return None
 
     @keep_session
+    def find_default_gwv6(self):
+        """
+        Find host default ipv6 gateway
+
+        :return: default gateway
+        :rtype: string
+        """
+        out = self._cmd(["ip", "-6", "route"]).splitlines()
+        for i in out:
+            if re.search("default", i):
+                default_gw = re.findall(
+                    r'(?<=\s)[0-9a-fA-F:]{3,}(?=\s)', i
+                )
+                if netaddr.valid_ipv6(default_gw[0]):
```

Comment:
I would prefer to keep it there, if the regex fails to match the address correctly.